docs/generate_genome_attributes.md

```diff
@@ -0,0 +1,24 @@
+# Generate Genome Attributes
+
+This document describes the steps and scripts used to generate attributes of a genome. 
```

Comment:
Can you document the folder structure for the data, similar to how we do it for the codebase in the readme?

Reply:
👍

Reply:
:+1: